src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/DesignTimeTagHelperWriter.cs

```diff
@@ -14,18 +14,9 @@ public class DesignTimeTagHelperWriter : TagHelperWriter
     {
         public string CreateTagHelperMethodName { get; set; } = "CreateTagHelper";
 
-        public override void WriteDeclareTagHelperFields(CSharpRenderingContext context, DeclareTagHelperFieldsIRNode node)
+        public override void WriteDeclareTagHelperFields(CSharpRenderingContext context)
```

Comment:
Still need this to write out the declaration and initialization of all the tag helper variables like `TagHelperExecutionContext`, `TagHelperRunner` etc. Should this be internal?
We can't make them be regular `FieldDeclarationIRNode`s because the type names used aren't known at IR time.